README.md

```diff
@@ -9,22 +9,25 @@ The Open-RMF platform for multi-fleet robot management.
 
 ---
 
-For specific rmf versions follow the instructions given below:
-
-## Installation Instructions
-
 These are the current Open-RMF binary releases available:
 
 | RMF Version | Installation Instructions                                                        | Supported distros                                    | Supported ROS2 versions |
 | ----------- | -------------------------------------------------------------------------------- | ---------------------------------------------------- | ----------------------- |
-| 21.09       |  [Installation instructions](https://github.com/open-rmf/rmf/tree/release/21.09) | Ubuntu 20.04, Ubuntu 21.09, RHEL 8 (deployment only) | Foxy, Galactic          |
+| 22.09       |  [Install RMF 22.09](https://github.com/open-rmf/rmf/tree/release/22.09) | Ubuntu 22.04, RHEL 8 (deployment only) | Humble          |
+| ~~21.09~~ (EOL) |  ~~[Install RMF 21.09](https://github.com/open-rmf/rmf/tree/release/21.09)~~ | ~~Ubuntu 20.04, Ubuntu 21.09, RHEL 8 (deployment only)~~ | ~~Foxy, Galactic~~        |
+
+## Installation Instructions
+
+Even though binary versions are avilable, given the fast development and frequent updates to Open-RMF,
+it is hevily recommended to follow instructions here and **install from
```

Comment:
```suggestion
it is heavily recommended to follow instructions here and **install from
```

Reply:
Do we want to link a section from  below when we say "here"?

Reply:
I had a link there to the "source installation" section but then I realized that might make people skip the installation of ROS and Gazebo before trying to install RMF from source so I removed it. Maybe reword the `here` into `succeeding` or `subsequent`?

Reply:
Maybe "follow instructions below and .."?

Reply:
Yup, sounds good, added.